apps/remix-ide/src/app/udapp/run-tab.js

```diff
@@ -78,7 +78,7 @@ export class RunTab extends ViewPlugin {
   }
 
   sendTransaction (tx) {
-    _paq.push(['trackEvent', 'udapp', 'sendTx'])
+    _paq.push(['trackEvent', 'udapp', 'send', 'transaction'])
```

Comment:
I think these names "send" and "transaction" are too generic - consider something like this:_paq.push(['trackEvent', 'udapp', 'sendTx', 'udappTransaction'])